client/src/codegen/go-client.ts

```diff
@@ -650,6 +650,24 @@ export class GoGenerator extends Generator {
     }
   }
 
+  printSecret(options: RenderOptions): string | null {
+    if (!options.secret) {
+      return null
```

Comment:
Wouldn't this branch result in us printing `var Secret = null`? That wouldn't be correct.